src/ceph-volume/ceph_volume/tests/functional/lvm/xenial/filestore/dmcrypt/group_vars/all

```diff
@@ -0,0 +1,32 @@
+---
+
+ceph_dev: True
+cluster: ceph
+public_network: "192.168.3.0/24"
+cluster_network: "192.168.4.0/24"
```

Comment:
I don't see ``dmcrypt: True`` set anywhere here.

Reply:
I was testing you